frontend/src/components/header/Header.js

```diff
@@ -0,0 +1,28 @@
+import styled from 'styled-components/macro'
+
+export default function Header() {
+  return (
+    <HeaderStyled>
+      <HeadingContainer>
+        <Heading>My Plants</Heading>
+      </HeadingContainer>
+    </HeaderStyled>
+  )
+}
+
+const HeaderStyled = styled.header`
+  display: flex;
+`
+
+const HeadingContainer = styled.div`
+  background: var(--primary-dark);
+  padding: 20px 30px 13px 20px;
+  border-radius: 0 0 50px 0;
+`
```

Comment:
I like how you defined each element individually and didn't nest anything! 👍 Very clean!